panels/user-accounts/um-app-permissions.c

```diff
@@ -286,6 +296,7 @@ update_oars_level (UmAppPermissions *self)
   const gchar *rating_age_category;
   guint maximum_age;
   gsize i;
+  gboolean all_categories_unset = TRUE;
```

Comment:
Nitpick: please move `all_categories_unset = TRUE;` to below `maximum_age = 0;` at line 303